test/smoke/src/helpers/setupEnvironmentHelper.ts

```diff
@@ -0,0 +1,220 @@
+        let expoCliCommand = process.platform === "win32" ? "expo-cli.cmd" : "expo-cli";
+        let installerProcess = cp.spawn(expoCliCommand, ["android"], {cwd: expoAppPath, stdio: "pipe"});
+        installerProcess.stdout.on("data", (data) => {
+            const string = filterProgressBarChars(data.toString().trim());
+            if (string !== "") {
+                console.log(`stdout: ${data.toString().trim()}`);
+            }
+        });
+        installerProcess.stderr.on("data", (data) => {
+            const string = filterProgressBarChars(data.toString().trim());
+            if (string !== "") {
+                console.error(`stderr: ${string}`);
+            }
+        });
+        installerProcess.on("close", () => {
+            console.log("*** expo-cli terminated");
+        });
+        installerProcess.on("error", (error) => {
+            console.log("Error occurred in expo-cli process: ", error);
+        });
+        await AndroidEmulatorHelper.checkIfAppIsInstalled(this.expoPackageName, 100 * 1000);
+        kill(installerProcess.pid, "SIGINT");
+        await sleep(1000);
+        AndroidEmulatorHelper.enableDrawPermitForApp(this.expoPackageName);
+    }
+
+    // Installs Expo app on iOS device via "expo install:ios" command
+    public static async installExpoAppOnIos(expoAppPath: string) {
+        return new Promise((resolve, reject) => {
+            console.log(`*** Installing Expo app on iOS simulator with 'expo-cli install:ios' command`);
+            let installerProcess = cp.spawn("expo-cli", ["install:ios"], {cwd: expoAppPath, stdio: "pipe"});
+            installerProcess.stdout.on("data", (data) => {
+                const string = filterProgressBarChars(data.toString().trim());
+                if (string !== "") {
+                    console.log(`stdout: ${string}`);
+                }
+            });
+            installerProcess.stderr.on("data", (data) => {
+                const string = filterProgressBarChars(data.toString().trim());
+                if (string !== "") {
+                    console.error(`stderr: ${string}`);
+                }
+            });
+            installerProcess.on("close", () => {
+                console.log("*** expo-cli terminated");
+                resolve();
+            });
+            installerProcess.on("error", (error) => {
+                console.log("Error occurred in expo-cli process: ", error);
+                reject(error);
+            });
+        });
+    }
+
+    public static addIosTargetToLaunchJson(workspacePath: string) {
+        let launchJsonPath = path.join(workspacePath, ".vscode", "launch.json");
+        console.log(`*** Implicitly adding target to "Debug iOS" config for ${launchJsonPath}`);
+        let content = JSON.parse(fs.readFileSync(launchJsonPath).toString());
+        let found = false;
+        for (let i = 0; i < content.configurations.length; i++) {
+            if (content.configurations[i].name === "Debug iOS") {
```

Comment:
Why this value is hardcoded? I think it would be better if this function will be more universal

Reply:
@SounD120 because highly unlikely it will be changed in future. It's debug configuration name.

Reply:
Yes, but what if we want to add device for such parameter for Android emulator in future? Or if we want to add any other parameters to debug configurations?

Reply:
> Yes, but what if we want to add device for such parameter for Android emulator in future? Or if we want to add any other parameters to debug configurations?

Then we will edit this line and another :)

Reply:
I'd like to add "to do" on this.